apps/web/src/features/ratings/components/emoji-rating-popover.tsx

```
@@ -480,7 +481,12 @@ export function EmojiRatingPopover({
           onClick={(e) => e.stopPropagation()}
         >
           <DrawerHeader className="p-6 pb-0">
-            <DrawerTitle asChild>{headerContent}</DrawerTitle>
+            <DrawerTitle>{existingRating ? 'update emoji rating' : 'rate with emoji'}</DrawerTitle>
```

ESLint

[error] Replace `{existingRating·?·'update·emoji·rating'·:·'rate·with·emoji'}` with `⏎··············{existingRating·?·'update·emoji·rating'·:·'rate·with·emoji'}⏎············`

(prettier/prettier)